changelog

```diff
@@ -1,3 +1,95 @@
+Version 0.9.11:
+  * Big changes:
+    * Added a new area, the Graveyard, a region of the Ember Waste inhabited by a new hostile alien faction. (@beccabunny)
```

Comment:
It may be more correct to say "a region *near* the Ember Waste"

Reply:
Well the Graveyard is part of the Ember Waste, isn't it?

Reply:
I consider it part of the Ember Waste, for whatever that's worth.

Reply:
Well, then I think it's fine